README.md

```diff
@@ -8,6 +8,12 @@
 
 [![Autotest Status](http://autotest.ardupilot.org/autotest-badge.svg)](http://autotest.ardupilot.org/)
 
+Ardupilot is the most advanced, full-featured and reliable open source autopilot software available. It has
+been developed over 5+ years by a team of diverse professional engineers and computer scientists. It is the
```

Comment:
Instead of the 5+ years time span, just mention that it is "under development since 2010"

Reply:
@amilcarlucas Would it make sense to also say that at http://ardupilot.org/about?